src/wristpy/processing/metrics.py

```diff
@@ -0,0 +1,30 @@
+"""Calculate base metrics, anglez and enmo."""
+
+import numpy as np
+
+from wristpy.core.models import Measurement
+
+
+def euclidean_norm_min_one(acceleration: Measurement) -> Measurement:
```

Comment:
Why not just name the function euclidean_norm_minus_one? It only saves two characters?

Reply:
Think that might've been my suggestion in case Freymon felt the name was getting _too_ long. Though that may have been my Dutch slipping through (Dutch word for minus is min), forgetting that min is more commonly used to abbreviate minimum. So yeah, I disagree with past Reinder :P